Diagnostics/AVTester/Test-ExchAVExclusions.ps1

```diff
@@ -370,7 +370,8 @@ while ($currentDiff -gt 0) {
         #Exchange 2019
         $ModuleAllowList.add("Microsoft.Exchange.BigFunnelFlow.28.dll")
         $ModuleAllowList.add("BigFunnel.NeuralTree.dll")
-
+        $ModuleAllowList.add("DocParserWrapper.dll")
+	
```

Comment:
It looks you have a tab character on line 374:
![image](https://github.com/user-attachments/assets/6fef38da-b192-4c17-a83b-984a73e0bc11)
Code Formating Scritp is complaining about this:
![image](https://github.com/user-attachments/assets/5296e9ba-3d3e-43bf-999b-a0afb6b75423)